readme.md

```diff
@@ -33,6 +36,14 @@ The destination folder is optional, and defaults to the current folder name. Wat
     apis      programs
     github readme.md
 
+#### Adding Authentication
```

Comment:
@eric-wieser feel free to give me feedback on the verbiage here... Have you played with GH's new code review stuff... its seems kind of cool. You can start a review instead of "Add single comment" and it will wait for you to finish commenting and submit all your comments as one message.